package.json

```diff
@@ -46,7 +46,8 @@
     "*.js": "eslint --cache --fix"
   },
   "resolutions": {
-    "ember-cli-htmlbars/semver": "~7.3.0"
+    "ember-cli-htmlbars/semver": "~7.3.0",
+    "ember-modifier": "3.2.7"
```

Comment:
Tests were failing with `ember-modifier@^4`. Pinning to v3 for now. Will open a GitHub issue for it. Some issues with `ember-modifier@^4` were reported in Discord channel as well.